### PR TITLE
Ess 2670 update drio python documentation to let the user know that cold path and adx has different time resolution

### PR DESCRIPTION
**Pull request by hanne-opseth-rygg-4ss**
### This PR is related to user story [ESS-2670](https://4insight.atlassian.net/jira/software/c/projects/ESS/boards/34?selectedIssue=ESS-2670)

[ESS-2670]: https://4insight.atlassian.net/browse/ESS-2670?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by hanne-opseth-rygg-4ss** [comment]
I will make sure when I release the package that the release notes will not become weird

File: docs/user_guide/manage_series.rst [hunk 1]
```diff
@@ -149,7 +149,9 @@ You can also access any data you have ``TimeSeriesId`` (and authorization) for w
 
 .. warning::
 
-    The time resolution of aggregated data is in milliseconds, while the time resolution of non-aggregated data is in nanoseconds. This may lead to discrepancies in data when comparing the two, and some datapoints might get lost when using aggregation to access data, in cases when there are multiple datapoints within the same nanosecond.
+
+    The time resolution of aggregated data is in 100 nanoseconds, while the time resolution of non-aggregated data is in nanoseconds. This may lead to discrepancies in data when comparing the two, and some datapoints might get lost when using aggregation to access data, in cases when there are multiple datapoints within the same 100 nanosecond range.
```
Comment (heidi-holm-4ss) [inline docs/user_guide/manage_series.rst:153]: maybe rephrase to "The time resolution of aggregated data is in ticks (1tick = 100 nanoseconds)..."

**Review by heidi-holm-4ss — DISMISSED** [review]
please see small comment

**Review by heidi-holm-4ss — APPROVED** [review]
